scripts/import_katu.sql

```diff
@@ -0,0 +1,30 @@
+set client_encoding = 'latin1';
+begin;
+
+create temporary table tmp_posti_baf(
+    data text
+) on commit drop;
+\copy tmp_posti_baf FROM '../data/posti/BAF_20210821.dat' DELIMITER AS '|'
+
+create temporary table tmp_posti_baf_formatted(
+    kuntakoodi char(3),
+    katunimi_fi text,
+    katunimi_sv text
+) on commit drop;
+
+INSERT INTO tmp_posti_baf_formatted(kuntakoodi, katunimi_fi, katunimi_sv)
+SELECT
+    substring(data, 214, 3) kuntakoodi,
+    nullif(trim(substring(data, 103, 30)), '') katunimi_fi,
+    nullif(trim(substring(data, 133, 30)), '') katunimi_sv
+FROM tmp_posti_baf
+;
+
+insert into jkr_osoite.katu(kunta_koodi, katunimi_fi, katunimi_sv)
```

Comment:
I think there is no need to import street names from other than the dvv data. Is this file unnecessary then?

Reply:
It might be unnecessary. I included it here and in the script file comments because the DVV streets names *might* contain too many errors, so it makes sense in my opinion to have a fallback script in the core to import the original error-free data from posti. We don't know yet if it is needed in the Lahti case, though.